osv/purl_helpers_test.py

```diff
@@ -64,6 +64,9 @@ def tests_package_to_purl(self):
     self.assertEqual('pkg:deb/debian/nginx?arch=source',
                      purl_helpers.package_to_purl('Debian', 'nginx'))
 
+    self.assertEqual('pkg:deb/echo/nginx?arch=source',
```

Comment:
Tests are failing here, getting `pkg:deb/echo/nginx` without arch=source.

On line 129 in purl_helpers we add arch=source to debian purls. Whether you want this depends on your own ecosystem (i.e. do you build for multiple architectures?, will your vulnerabilties apply to specific architectures.)

The easiest option here is probably just to remove arch=source here.

Reply:
I removed the arch=source, it's un needed here.